openedx/features/course_experience/templates/course_experience/course-sock-fragment.html

```diff
@@ -0,0 +1,69 @@
+## mako
+
+<%page expression_filter="h"/>
+<%namespace name='static' file='../static_content.html'/>
+
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML
+%>
+
+<%block name="content">
+    % if show_course_sock:
+    <div class="verification-sock">
+        <button type="button" class="btn btn-brand focusable action-toggle-verification-sock">
+            ${_("Learn About Verified Certificate")}
+        </button>
+        <div class="verification-main-panel">
+            <div class="verification-desc-panel content-main">
+                <h2>${_("edX Verified Certificate")}</h2>
```

Comment:
@nasthagiri how are you handling the marketing material in your experiment? Is there anything we could learn from that?